src/main/java/org/tdl/vireo/model/repo/impl/SubmissionRepoImpl.java

```diff
@@ -467,7 +471,31 @@ public int compare(SubmissionListColumn svc1, SubmissionListColumn svc2) {
                 switch (String.join(".", submissionListColumn.getValuePath())) {
                 case "fieldValues.value":
 
-                    Long predicateId = fieldPredicateRepo.findByValue(submissionListColumn.getPredicate()).getId();
+                    FieldPredicate fieldPredicate = fieldPredicateRepo.findByValue(submissionListColumn.getPredicate());
+
+                    // if the predicate is not found, see if it is space delimited set of predicates
+                    // for each predicate, add a submission list column clone with predicate and sort order updated
+                    if (fieldPredicate == null) {
+                        String[] predicates = submissionListColumn.getPredicate().split(" ");
```

Comment:
This seems risky.
What about single predicates that have spaces in their name?

Reply:
Yes, it is risky and potential for breaking. I will explore some alternatives.

Reply:
An improved design will likely require a domain model addition and additional logic to accomplish. Not sure how we can make this safer without a redesign. I propose an issue be created to explore another approach.